Difi.Felles.Utility/CertificateValidator.cs

```diff
@@ -27,18 +67,63 @@ public static CertificateValidationResult ValidateCertificateAndChain(X509Certif
             return certificateChainValidator.Validate(certificate);
         }
 
+        /// <summary>
+        ///     Validates the certificate itself. Validates that certificate is
+        ///     <list type="bullet">
+        ///         <item> Not null </item>
+        ///         <item> Issued to organization number </item>
+        ///         <item> Is activated </item>
+        ///         <item> Not Expired </item>
+        ///     </list>
+        /// </summary>
+        /// <remarks>
+        ///     Does not validate the certificate chain. Please use <see cref="ValidateCertificateAndChain" /> for including
+        ///     chain validation
+        /// </remarks>
+        /// <param name="certificate">The certificate to validate</param>
+        /// <param name="certificateOrganizationNumber">The organization number the certificate is issued to</param>
+        /// <returns>The result of the certificate validation</returns>
```

Comment:
Denne metoden er den eneste som beskriver hva som returneres. Bør det være slik på alle, eller trengs det ikke på noen? Litt rart når det bare er slik for én i hvert fall :)